tox.ini

```diff
@@ -97,3 +97,5 @@ sitepackages = {env:SITE_PACKAGES:false}
 suicide_timeout = 60.0
 interrupt_timeout = 60.0
 terminate_timeout = 60.0
+
+[testenv:requirements]
```

Comment:
This is needed to inform tox that there is an environment named `requirements`. Otherwise `tox -e requirements` won't work. If we had any excuse to mention the `requirements` env elsewhere in `tox.ini` (to set any envvars for it, install any dependencies, or give it any default commands to run) then we wouldn't need this.